caffe-grid/src/test/python/PythonTest.sh

```diff
@@ -0,0 +1,5 @@
+export CAFFE_ON_SPARK=$(pwd)
+export LD_LIBRARY_PATH=${CAFFE_ON_SPARK}/caffe-public/distribute/lib:${CAFFE_ON_SPARK}/caffe-distri/distribute/lib:/usr/lib64:/lib64
```

Comment:
This definition doesn't include /usr/local/lib. Why don't you change it to:

export LD_LIBRARY_PATH=${CAFFE_ON_SPARK}/caffe-public/distribute/lib:${CAFFE_ON_SPARK}/caffe-distri/distribute/lib:/usr/lib64:/lib64:${LD_LIBRARY_PATH}